bazel/README.md

```diff
@@ -112,6 +112,28 @@ for how to update or override dependencies.
     The full version of Xcode (not just Command Line Tools) is also required to build Envoy on macOS.
     Envoy compiles and passes tests with the version of clang installed by Xcode 11.1:
     Apple clang version 11.0.0 (clang-1100.0.33.8).
+    
+    #### trouble shooting
```

Comment:
`Troubleshoot` should be a single word. If the spell checker complains, you should add the word to the words database.